### PR TITLE
#81969 KOMBIT VA-119 - New CanImpersonate attribute for Anvendersyste…

### PR DESCRIPTION
**Pull request by ctthang**
…m - Provisioning service

### CODE REVIEW

File: interface/STS-Provision.json [hunk 1]
```diff
@@ -490,112 +490,33 @@
...
+            "required": true,
+            "type": "string"
           },
           {
-            "name": "body",
-            "in": "body",
-            "description": "The ImpersonationAssignment to delete",
+            "name": "value",
+            "in": "path",
+            "description": "true/false to specifies if this Anvendersystem can impersonate other IT systems",
```
Comment (thuannguy) [inline interface/STS-Provision.json]: "to specify"

File: interface/STS-Provision.json [hunk 2]
```diff
@@ -1932,12 +1853,9 @@
...
         },
-        "ImpersonationAssignments": {
-          "description": "A list of impersonation assignments assigned to this Anvendersystem (service consumer).",
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/ImpersonationAssignment"
-          }
+        "CanImpersonate": {
+          "type": "boolean",
+          "description": "Specifies if STS will allow OnBehalfOf for all requests from this Anvendersystem (user system)"
```
Comment (thuannguy) [inline interface/STS-Provision.json]: I would suggest 

"description": "Specifies if this Anvendersystem can impersonate other systems."

File: interface/STS-Provision.json (hunk 1 above)
Comment (ctthang) [reply interface/STS-Provision.json]: updated

File: interface/STS-Provision.json (hunk 2 above)
Comment (ctthang) [reply interface/STS-Provision.json]: updated

**Review by thuannguy — APPROVED** [review]

**Pull request merged**